core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java

```diff
@@ -640,10 +640,10 @@ public Result visit(Values e) {
         // In case of empty values, we need to build:
         // select * from VALUES(NULL, NULL ...) as T (C1, C2 ...)
         // where 1=0.
-        List<SqlNode> nulls = IntStream.range(0, fieldNames.size())
-            .mapToObj(i ->
-                SqlLiteral.createNull(POS)).collect(Collectors.toList());
-        selects.add(ANONYMOUS_ROW.createCall(new SqlNodeList(nulls, POS)));
+        SqlNodeList nulls = IntStream.range(0, fieldNames.size())
+            .mapToObj(i -> SqlLiteral.createNull(POS))
+            .collect(SqlNode.toList());
```

Comment:
What do you think of `Collections.nCopies(nCopies, SqlLiteral.createNull(POS))`?

Reply:
Fine. I didn't create that code, just reformatted it.